### PR TITLE
added template for Go

### PR DESCRIPTION
**Pull request by nahidhasan98**
created go.go

### CODE REVIEW

File: 1000/go.go [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+	}
+}
+
+/**
+ * Returns the total number of problems.
+ *
+ * @param a denotes the number of problems in the first computer
+ * @param b denotes the number of problems in the second computer
+**/
+func process(a, b int) int {
```
Comment (faiyaz26) [inline 1000/go.go]: Move this method before the main method.

Comment (nahidhasan98) [reply 1000/go.go]: Done. Please check that out.

**Comment by faiyaz26** [comment]
Thanks for you contribution

**Comment by nahidhasan98** [comment]
I am glad. This was my first contribution. Thank you very much.

One more thing. Should I create a new branch for every problem dir/language?

**Comment by faiyaz26** [comment]
You are creating branch on your fork, so it doesn't matter whether you create another branch or not. Good practice is creating in another branch, in case you want to contribute multiple code (different problem) in the same time. 

Create pull request for one problem containing as much code (multiple language) possible

**Comment by nahidhasan98** [comment]
Okay. Got it now. Then I am closing the recent pull request for 1001.

**Pull request merged**